src/components/LocalIdentityModal/LocalIdentityModal.js

```diff
@@ -161,9 +155,7 @@ function Label(props) {
         display: block;
         margin: 20px 0;
         color: ${theme.surfaceContentSecondary};
-        ${textStyle('label2')}
-
-        & > div {
+        ${textStyle('label2')} & > div {
```

Comment:
These two should be on separate lines; it looks a bit confusing otherwise

Reply:
This is how `prettier` formats this because a `;` seems to be missing

Reply:
You can add another `;` when needed, styled-components is fine with it:

```jsx
         ${textStyle('label2')};
         & > div { 
```

Reply:
Oh sure, I was just stating what was happening but the fix already submitted another `;`